packages/core/src/scss/components/checkout/checkoutRemote/_checkoutRemote.scss

```diff
@@ -19,6 +19,7 @@
         width: remCalc(200px);
         #amazonpayCheckoutButton > div{
             width: 100% !important;
+            height: 51px !important;
```

Comment:
Is this button for top of checkout page? I believe we want to have a specific height for all the buttons.

Reply:
Yes, this is for AmazonPay button in Customer step. In this place all buttons have height set to 36px (rectangular yellow part of AmazonPay button too). Recently we changed AmazonPay button design to one with microtext and as I know it was Amazon demand. Previously in all places AmazonPay button had design without microtext. 
We need this change to move microtext a little bit down from the rectangular part because without it the microtext is overflowing it.